packages/auth-server/package.json

```diff
@@ -17,6 +17,9 @@
     "@vueuse/nuxt": "^11.1.0",
     "@wagmi/core": "^2.13.3",
     "@wagmi/vue": "^0.0.49",
+    "axios": "^1.7.9",
+    "jsonwebtoken": "^9.0.2",
+    "jwk-to-pem": "^2.0.7",
```

Comment:
Have you checked `jose`? It's a library created by auth0 that I believe it's kind of the standard for jwt manipulation.

With jose you can use directly the JWK instead of having to convert to PEM:


https://github.com/panva/jose/blob/HEAD/docs/jwt/verify/functions/jwtVerify.md
https://github.com/panva/jose/blob/HEAD/docs/key/import/functions/importJWK.md